src/invoice.rs

```diff
@@ -87,6 +146,18 @@ pub struct RgbInvoice {
     pub unknown_query: IndexMap<String, String>,
 }
 
+#[derive(Clone, PartialEq, Eq, Debug, Display, Error, From)]
+#[display(inner)]
+pub enum TransportParseError {
+    #[display(doc_comments)]
+    /// invalid transport {0}.
+    InvalidTransport(String),
+
+    #[display(doc_comments)]
+    /// invalid transport host {0}.
+    InvalidTransportHost(String),
+}
+
 #[derive(Clone, PartialEq, Eq, Debug, Display, Error, From)]
 #[display(inner)]
```

Comment:
Nit: we ca do
```suggestion
#[display(doc_comments)]
```

and remove all `doc_comments` in the variants - adding `#[display(inner)]` where they are absent